app/src/main/java/com/codefororlando/petadoption/helper/ContactFormatter.kt

```diff
@@ -0,0 +1,20 @@
+package com.codefororlando.petadoption.helper
+
+class ContactFormatter {
+
+    companion object {
+
+        fun formatPhoneNumber(unformattedNumber: String): String {
```

Comment:
If you annotate these functions with `@JvmStatic`, you won't need to call them like this in Java: `ContactFormatter.Companion.formatPhoneNumber();`, you can just use `ContactFormatter.formatPhoneNumber();`